jest.config.js

```diff
@@ -30,11 +30,7 @@ module.exports = {
 		'node',
 	],
 	moduleNameMapper: {
-		'@wordpress/blocks': '<rootDir>/gutenberg/blocks',
-		'@wordpress/editor': '<rootDir>/gutenberg/editor',
-		'@wordpress/element': '<rootDir>/gutenberg/element',
-		'@wordpress/hooks': '<rootDir>/wordpress/hooks',
-		'@wordpress/utils': '<rootDir>/gutenberg/utils',
```

Comment:
I wonder if we need to remove the relevant mapping in [the Flow config](https://github.com/wordpress-mobile/gutenberg-mobile/blob/update/run-ios/.flowconfig#L86) as well. WDYT?

Reply:
I wasn't aware, we don't use it so we should remove.